client/blocks/eligibility-warnings/excessive-disk-space.tsx

```diff
@@ -0,0 +1,20 @@
+import { localize, LocalizeProps } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import PlanStorage from 'calypso/blocks/plan-storage';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+
+const ExcessiveDiskSpace = ( { translate }: { translate: LocalizeProps[ 'translate' ] } ) => {
+	const selectedSiteId = useSelector( getSelectedSiteId );
+	return (
+		<div>
+			{ translate( "Your site's disk usage exceeds the amount provided by its plan." ) }
+			<div className="eligibility-warnings__plan-storage-wrapper">
+				<PlanStorage siteId={ selectedSiteId }>{ null }</PlanStorage>
+			</div>
+			{ translate(
+				'Please purchase a plan with additional storage or contact our support team for help.'
```

Comment:
I did this, but [added two plan checks](https://github.com/Automattic/wp-calypso/blob/54425b90730d37be961c94af3d6f799ebbda4189/client/blocks/eligibility-warnings/excessive-disk-space.tsx#L18-L19).  :(
In theory, they should be temporary since addons are coming soon?